src/gtk/settings.cpp

```diff
@@ -24,7 +24,7 @@
 #include "wx/gtk/private/win_gtk.h"
 #include "wx/gtk/private/gtk2-compat.h"
 
-bool wxGetFrameExtents(GdkWindow* window, int* left, int* right, int* top, int* bottom);
+bool wxGetFrameExtents(GdkWindow* window, wxTopLevelWindow::DecorSize* decorSize);
```

Comment:
Sure, it was only a consistency observation. It could be the other way around too, it's just a bit confusing to see both qualifiers used for the same type.